fuzz/meson.build

```diff
@@ -3,12 +3,23 @@ fuzz_src = files(
     'writeout.c',
 )
 
+link_language = 'c'
+fuzz_ldflags = get_option('fuzz-ldflags')
+# Assume that if fuzz_ldflags is set, C++ linking is required.
+# This is necessary for OSS-Fuzz.
+if fuzz_ldflags != ''
+    add_languages('cpp', native: false)
+    link_language = 'cpp'
```

Comment:
nit: `fuzz_link_langauge` to make it clearer this only applies to the fuzzer binary